src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/xsv/LocatableXsvFuncotationFactory.java

```diff
@@ -71,35 +72,43 @@
     //==================================================================================================================
     // Constructors:
 
-    public LocatableXsvFuncotationFactory(){
+    @VisibleForTesting
+    LocatableXsvFuncotationFactory(){
         this(DEFAULT_NAME, DEFAULT_VERSION_STRING);
     }
 
-    public LocatableXsvFuncotationFactory(final String name, final String version){
-        this(name, version, new LinkedHashMap<>());
-    }
+    @VisibleForTesting
+    LocatableXsvFuncotationFactory(final String name, final String version, final List<String> supportedFields){
 
-    public LocatableXsvFuncotationFactory(final String name, final String version, final LinkedHashMap<String, String> annotationOverridesMap){
         this.name = name;
         this.version = version;
 
-        this.annotationOverrideMap = new LinkedHashMap<>(annotationOverridesMap);
+        supportedFieldNames = new LinkedHashSet<>(supportedFields);
+        initializeFieldNameLists();
     }
 
     @VisibleForTesting
```

Comment:
Again, I question the need for all these visible-for-testing constructors -- why can't you use the regular constructors in your tests?

Reply:
Fixed.